waspc/cli/src/Wasp/Cli/Command/News/Display.hs

```diff
@@ -0,0 +1,37 @@
+{-# LANGUAGE OverloadedRecordDot #-}
+
+module Wasp.Cli.Command.News.Display (showNewsEntry) where
+
+import Data.List (intercalate)
+import Data.Time (defaultTimeLocale, formatTime)
+import Wasp.Cli.Command.News.Core (NewsEntry (..), NewsLevel (..))
+import Wasp.Util (indent, wrapString)
+import qualified Wasp.Util.Terminal as Term
+
+showNewsEntry :: NewsEntry -> String
+showNewsEntry newsEntry =
+  intercalate "\n" [headerLine, levelLine, bodyBlock]
+  where
+    headerLine =
+      Term.applyStyles [Term.Bold] newsEntry.title
+        <> " "
+        <> Term.applyStyles [Term.Bold] (replicate dotCount '.')
+        <> " "
+        <> Term.applyStyles [Term.Yellow, Term.Bold] date
+
+    levelLine = showNewsLevel newsEntry.level
+
+    bodyBlock = Term.applyStyles [Term.Grey] $ indent indentSize $ wrapString (maxColumns - indentSize) newsEntry.body
+
+    dotCount = max minDotsCount (maxColumns - length newsEntry.title - length date - 2)
```

Comment:
Why minus 2? As I said above in the comment, you should either explain this is caused by the two spaces around the dot separator, which is fragile but at least there is a comment, or better, you should write a function that adds repeated separator between the two string, and repeats it till it fills the length.
Check the earlier comment, I think you maybe missed it because you never commented on it.